core/boot.js

```diff
@@ -297,7 +302,7 @@ async function initBackgroundServices({config}) {
 
  * @returns {Promise<object>} ghostServer
  */
-async function bootGhost() {
+async function bootGhost({withBackend = true, withFrontend = true} = {}) {
```

Comment:
Also a minor comment, but these params could simply be `backend` and `frontend`. With makes it more confusing IMO, not less 😉